express-api/app-data-source.ts

```diff
@@ -0,0 +1,20 @@
+import { DataSource } from 'typeorm';
+import { config } from 'dotenv';
+import { resolve } from 'path';
+
+config({ path: resolve(__dirname, '../.env') });
+
+export const AppDataSource = new DataSource({
+  type: 'postgres',
+  host: 'localhost',
+  port: Number(parseInt(process.env.DATABASEPORT)),
+  username: process.env.POSTGRES_USER,
+  password: process.env.POSTGRES_PASSWORD,
+  database: process.env.POSTGRES_DB,
+  synchronize: true,
```

Comment:
Also wondering if this is safe to have on. It says not to be used for production, as it could erase data. Might be better to keep it off.

Reply:
I was thinking that we could leave it on until the entities are set up - otherwise we would have to create all entities through migrations.